### ISSUE TITLE
Documentation: Adding status to cards

### ISSUE THREAD

**Issue by danbrady** [department-of-veterans-affairs/vets-design-system-documentation#3723]
## Description

Based on research by the Ask VA team, we need to update guidance to add the proper way to use `va-tag` to indicate status in `va-card`.

The proposed solution should have the header announce "status" by screen readers, then the visible status text, and then the header text. 

Related documentation:
- [Designs with proposed code snippet](https://www.figma.com/design/aQ6JsjD4pvMxSVPAZHllMX/AVA-Page-Library?node-id=1224-133216&t=wsp4ZNmNqsgJcO8R-1) (Figma)
- [Ask VA form flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1708479170047/8ebb517a213e05ebc500087f5661a993a41392e8?sender=uaa6c41f47fccd86fc83a1520) (Mural)
- [Assistive Technology Research](https://dsva.slack.com/archives/C0NGDDXME/p1733770425384109?thread_ts=1733752805.986509&cid=C0NGDDXME) PDF of slides from presentation
- [Slack conversation](https://dsva.slack.com/archives/C0NGDDXME/p1733769138571939?thread_ts=1733752805.986509&cid=C0NGDDXME)
- [Ask VA ticket](https://github.com/department-of-veterans-affairs/ask-va/issues/1552) that requests code update

## Tasks
- [ ] Card guidance updated to include how to correctly add a status to a card component. 

## Acceptance Criteria
- [ ] Updated guidance and code snippet reviewed by accessibility specialist.
- [ ] Card guidance updated to include how to correctly add a status to a card component. 

CC: @t-michaud
